.editorconfig

```diff
@@ -8,4 +8,4 @@ indent_brace_style = K&R
 
 [*.py]
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
@robertnishihara I changed this because my IDE was always adding 2 spaces. We can revert if necessary, but the project uses 4 spaces.

Reply:
quick question - why do we need this in the codebase?

Reply:
I don't think we do.

Reply:
This looks the right change, thanks.

I also agree taht we don't need it in the codebase.